### PR TITLE
siegfried: Update to 1.8.0 version

### PR DESCRIPTION
**Pull request by mamedin**
* Build version v1.8.0-1, with custom archivematica configuration
* (Ubuntu) Use DH_GOPKG=github.com/richardlehane/siegfried/cmd...
    (See https://github.com/richardlehane/siegfried/pull/98)
* Use Go 1.13

The `DH_GOPKG=github.com/richardlehane/siegfried/cmd...` was
already used on v1.7 CentOS package.

Connects to https://github.com/archivematica/Issues/issues/1075

### CODE REVIEW

**Comment by mamedin** [comment]
I updated go to 1.13 because:

```
artefactual@am110xenial:~/src/siegfried$ go install -tags archivematica -v github.com/richardlehane/siegfried/cmd/... | grep -i "requires Go"
github.com/richardlehane/siegfried/internal/persist
go build github.com/richardlehane/siegfried/internal/persist: module requires Go 1.13
golang.org/x/sys/unix
go build golang.org/x/sys/unix: module requires Go 1.12
github.com/richardlehane/siegfried/internal/checksum
go build github.com/richardlehane/siegfried/internal/checksum: module requires Go 1.13
golang.org/x/image/riff
go build golang.org/x/image/riff: module requires Go 1.12
github.com/richardlehane/xmldetect
go build github.com/richardlehane/xmldetect: module requires Go 1.12
github.com/richardlehane/siegfried/pkg/config
go build github.com/richardlehane/siegfried/pkg/config: module requires Go 1.13
github.com/richardlehane/siegfried/pkg/loc/internal/mappings
go build github.com/richardlehane/siegfried/pkg/loc/internal/mappings: module requires Go 1.13
github.com/richardlehane/siegfried/pkg/mimeinfo/internal/mappings
go build github.com/richardlehane/siegfried/pkg/mimeinfo/internal/mappings: module requires Go 1.13
github.com/richardlehane/siegfried/pkg/pronom/internal/mappings
go build github.com/richardlehane/siegfried/pkg/pronom/internal/mappings: module requires Go 1.13
github.com/richardlehane/siegfried/internal/chart
go build github.com/richardlehane/siegfried/internal/chart: module requires Go 1.13
```

File: debs/siegfried/debian/rules [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 export DH_OPTIONS
 
-export DH_GOPKG := github.com/richardlehane/siegfried
+export DH_GOPKG := github.com/richardlehane/siegfried/cmd...
```
Comment (ross-spencer) [inline debs/siegfried/debian/rules]: From a go perspective the binary would be built from the `cmd` dir here I believe, but what is the reason behind this change in the package this time around? (Curiosity more than anything).

**Review by ross-spencer — COMMENTED** [review]
Just left a comment, all looks sensible to me. I'm perhaps not the most knowledgeable to approve though.

File: debs/siegfried/debian/rules (hunk 1 above)
Comment (ablwr) [reply debs/siegfried/debian/rules]: https://github.com/artefactual-labs/am-packbuild/pull/251#issuecomment-578112922

Comment (ross-spencer) [reply debs/siegfried/debian/rules]: To be clear I don't feel this is an issue, I was just wondering technically why so that I can learn, but that comment (which is just a stack trace) describes why we updated `Go` not why we changed this path, unless I am missing something?

Comment (sevein) [reply debs/siegfried/debian/rules]: I think that `DH_GOPKG := github.com/richardlehane/siegfried` was only working because of the way that golang.pm was building the target for `go install`. That probably stopped working when Richard adopted modules in Siegfried.

@mamedin, I think that golang.pm would prefer `github.com/richardlehane/siegfried/cmd` instead of `github.com/richardlehane/siegfried/cmd...`.

The discovery process would still work, e.g the build command results the same:

    go install -v -tags archivematica github.com/richardlehane/siegfried/cmd/roy github.com/richardlehane/siegfried/cmd/sf

But `dh_auto_install` names the temporary directory as `cmd` instead of `cmd...`. The latter sort of tells you that golang.pm isn't expecting that path.

**Review by sevein — APPROVED** [review]
I've suggested a small change but otherwise this is looking good. The binary built successfully and dpkg seems pretty happy with it.

File: debs/siegfried/debian/rules (hunk 1 above)
Comment (sevein) [reply debs/siegfried/debian/rules]: @ross-spencer FYI golang.pm is the thing that Debian uses to build Go packages. We have our own copy here in this repo (https://github.com/artefactual-labs/am-packbuild/blob/stable/1.10.x/debs/siegfried/files/golang.pm) because we need to build with `-tags=archivematica` (https://github.com/richardlehane/siegfried/blob/master/pkg/config/archivematica.go).

**Comment by mamedin** [comment]
Thanks @sevein , I just changed it. Merging!
